src/store/reducer.ts

```diff
@@ -1,20 +1,35 @@
-import { createReducer } from '@reduxjs/toolkit';
-import { changeCity, setOffers } from './action';
-import { DEFAULT_CITY } from '../consts';
+import { createSlice } from '@reduxjs/toolkit';
+import { DEFAULT_CITY, LoadingStatus } from '../consts';
 import { InitialStateType } from '../types';
+import { fetchGet } from './api-action';
 
 const initialState: InitialStateType = {
   city: DEFAULT_CITY,
-  offers: []
+  offers: [],
+  status: undefined
 };
 
-export const reducer = createReducer(initialState, (builder) => {
-  builder
-    .addCase(changeCity, (state, action) => {
-      state.city = action.payload;
-    })
-    .addCase(setOffers, (state, action) => {
+export const offersSlice = createSlice({
+  name: 'offers',
+  initialState,
+  reducers: {
+    setOffers: (state, action) => {
```

Comment:
https://redux-toolkit.js.org/tutorials/typescript#define-slice-state-and-action-types